2025-06-18 08:54:32,573 - ultibot_backend - INFO - [MainThread] - main.py:65 - Logging configurado exitosamente usando dictConfig.
2025-06-18 08:54:32,573 - ultibot_backend - INFO - [MainThread] - main.py:136 - Registrando routers de la API...
2025-06-18 08:54:32,585 - ultibot_backend - INFO - [MainThread] - main.py:148 - Todos los routers han sido registrados.
2025-06-18 08:54:32,585 - uvicorn.error - INFO - [MainThread] - server.py:82 - Started server process [27832]
2025-06-18 08:54:32,593 - uvicorn.error - INFO - [MainThread] - on.py:48 - Waiting for application startup.
2025-06-18 08:54:32,593 - ultibot_backend - INFO - [MainThread] - main.py:75 - Iniciando UltiBot Backend...
2025-06-18 08:54:32,593 - ultibot_backend.dependencies - INFO - [MainThread] - dependencies.py:48 - Initializing dependency container...
2025-06-18 08:54:32,593 - httpx - DEBUG - [MainThread] - _config.py:82 - load_ssl_context verify=True cert=None trust_env=True http2=False
2025-06-18 08:54:32,593 - httpx - DEBUG - [MainThread] - _config.py:148 - load_verify_locations cafile='C:\\Users\\zamor\\UltiBotInversiones\\.venv\\Lib\\site-packages\\certifi\\cacert.pem'
2025-06-18 08:54:32,773 - aiosqlite - DEBUG - [Thread-1] - core.py:104 - executing <function connect.<locals>.connector at 0x0000029BDEBCAAC0>
2025-06-18 08:54:32,773 - aiosqlite - DEBUG - [Thread-1] - core.py:106 - operation <function connect.<locals>.connector at 0x0000029BDEBCAAC0> completed
2025-06-18 08:54:32,773 - ultibot_backend.adapters.persistence_service - INFO - [MainThread] - persistence_service.py:93 - Conectado a SQLite en: ./ultibot_local.db.
2025-06-18 08:54:32,773 - httpx - DEBUG - [MainThread] - _config.py:82 - load_ssl_context verify=True cert=None trust_env=True http2=False
2025-06-18 08:54:32,773 - httpx - DEBUG - [MainThread] - _config.py:148 - load_verify_locations cafile='C:\\Users\\zamor\\UltiBotInversiones\\.venv\\Lib\\site-packages\\certifi\\cacert.pem'
2025-06-18 08:54:32,960 - ultibot_backend.services.order_execution_service - INFO - [MainThread] - order_execution_service.py:110 - Paper Trading Service inicializado con capital virtual: 10000.0 USDT
2025-06-18 08:54:32,962 - ultibot_backend.services.credential_service - INFO - [MainThread] - credential_service.py:28 - CredentialService __init__ intentando obtener encryption_key de ENV: Presente
2025-06-18 08:54:32,962 - ultibot_backend.services.credential_service - INFO - [MainThread] - credential_service.py:34 - Pasando la siguiente clave (str) a Fernet: m-28N... (truncada por seguridad)
2025-06-18 08:54:32,962 - ultibot_backend.services.unified_order_execution_service - INFO - [MainThread] - unified_order_execution_service.py:27 - UnifiedOrderExecutionService initialized with both real and paper execution services
2025-06-18 08:54:32,983 - ultibot_backend.services.ai_orchestrator_service - INFO - [MainThread] - ai_orchestrator_service.py:143 - AIOrchestrator initialized with gemini-1.5-pro-latest and PydanticOutputParser.
2025-06-18 08:54:32,983 - ultibot_backend.dependencies - INFO - [MainThread] - dependencies.py:127 - Dependency container initialized successfully.
2025-06-18 08:54:32,983 - ultibot_backend - INFO - [MainThread] - main.py:83 - Contenedor de dependencias inicializado.
2025-06-18 08:54:32,983 - ultibot_backend - INFO - [MainThread] - main.py:89 - Aplicaci√≥n iniciada correctamente.
2025-06-18 08:54:32,983 - uvicorn.error - INFO - [MainThread] - on.py:62 - Application startup complete.
2025-06-18 08:54:41,299 - ultibot_backend - INFO - [MainThread] - main.py:105 - Middleware: Solicitud entrante: GET /api/v1/config
2025-06-18 08:54:41,299 - ultibot_backend - INFO - [MainThread] - main.py:107 - Middleware: Solicitud procesada: GET /api/v1/config - Status: 200
2025-06-18 08:54:41,301 - uvicorn.access - INFO - [MainThread] - httptools_impl.py:481 - 127.0.0.1:54603 - "GET /api/v1/config HTTP/1.1" 200
2025-06-18 08:57:40,549 - ultibot_backend - INFO - [MainThread] - main.py:105 - Middleware: Solicitud entrante: GET /api/v1/config
2025-06-18 08:57:40,550 - ultibot_backend - INFO - [MainThread] - main.py:107 - Middleware: Solicitud procesada: GET /api/v1/config - Status: 200
2025-06-18 08:57:40,551 - uvicorn.access - INFO - [MainThread] - httptools_impl.py:481 - 127.0.0.1:54694 - "GET /api/v1/config HTTP/1.1" 200
2025-06-18 08:59:44,688 - ultibot_backend - INFO - [MainThread] - main.py:105 - Middleware: Solicitud entrante: GET /api/v1/config
2025-06-18 08:59:44,688 - ultibot_backend - INFO - [MainThread] - main.py:107 - Middleware: Solicitud procesada: GET /api/v1/config - Status: 200
2025-06-18 08:59:44,689 - uvicorn.access - INFO - [MainThread] - httptools_impl.py:481 - 127.0.0.1:54775 - "GET /api/v1/config HTTP/1.1" 200
2025-06-18 09:01:09,909 - ultibot_backend - INFO - [MainThread] - main.py:105 - Middleware: Solicitud entrante: GET /api/v1/config
2025-06-18 09:01:09,909 - ultibot_backend - INFO - [MainThread] - main.py:107 - Middleware: Solicitud procesada: GET /api/v1/config - Status: 200
2025-06-18 09:01:09,910 - uvicorn.access - INFO - [MainThread] - httptools_impl.py:481 - 127.0.0.1:54808 - "GET /api/v1/config HTTP/1.1" 200
